src/rajAJun20/FirstDifferIndex.java

```diff
@@ -0,0 +1,25 @@
+package rajAJun20;
+
+public class FirstDifferIndex {
+	
+	void differFirstIndexOfArrays(int[] arr1 , int[] arr2 ) {
+		
+		if(arr1.length == arr2.length) {
+				for(int index = 0 ; index < arr1.length ; index++ ) {
+					if(arr1[index] != arr2[index]) {
+						System.out.println("Numbers of both the arrays are not matching at index : " + index);
+					}
```

Comment:
Where is the return statement ??? differFirstIndexOfArrays() we are saying on first index what if other index value is also different.

Reply:
Used break now